stock_request_analytic/tests/test_stock_request_analytic.py

```diff
@@ -54,6 +55,7 @@ def prepare_order_request_analytic(self, aa, company):
                 'product_uom_id': self.pizza.uom_id.id,
                 'product_uom_qty': 5.0,
                 'analytic_account_id': aa.id,
+                'analytic_tag_ids': [(4, ati) for ati in ats_ids],
```

Comment:
More significant variable names and code readability

```suggestion
                'analytic_tag_ids': [(4, tag.id) for tag in analytic_tags],
```